make/autoconf/toolchain.m4

```diff
@@ -533,6 +533,8 @@ AC_DEFUN([TOOLCHAIN_EXTRACT_LD_VERSION],
     #   GNU ld (GNU Binutils for Ubuntu) 2.26.1
 
     LINKER_VERSION_STRING=`$LINKER -Wl,-v 2>&1 | $HEAD -n 1`
+    # Remove redundant a.out generated by clang, to workaound clang bug https://github.com/llvm/llvm-project/issues/132388
```

Comment:
Spelling and line length, otherwise this looks ok.
```suggestion
    # Remove redundant a.out generated by clang, to work around clang bug
    # https://github.com/llvm/llvm-project/issues/132388
```

Reply:
Thanks. PR has been updated.